agent/src/agent/source/validator.py

```diff
@@ -249,6 +251,26 @@ def validate_token(self):
         pass
 
 
+class VictoriaMetricsValidator(Validator):
+    VALIDATION_SCHEMA_FILE = 'victoria.json'
+
+    def validate_connection(self):
+        url = self.source.config['url'] + '/api/v1/export?match[]={__name__="not_existing_dsger43"}'
+        session = requests.Session()
+        if self.source.config[source.VictoriaMetricsSource.USERNAME]:
+            session.auth = (
+                self.source.config[source.VictoriaMetricsSource.USERNAME],
+                self.source.config[source.VictoriaMetricsSource.PASSWORD]
+            )
+        try:
+            res = session.get(url)
+            res.raise_for_status()
+        except requests.exceptions.RequestException:
+            raise ValidationException(
```

Comment:
It's better to add what exception exactly was raised (with error message) so we could understand the problem